### ISSUE TITLE
profile pictures not appearing for github enterprise

### ISSUE THREAD

**Issue by gzahavi** [desktop/desktop#11898]
### Describe the bug

i'm using github enterprise. in all of the commits and other places on github desktop, the profile pictures show up as the default gravatar icon, instead of the profile pictures as set on the enterprise server

in Desktop:
<img src="https://user-images.githubusercontent.com/81637465/113015651-c1818e00-914b-11eb-8692-075026119acb.png" width="30%" />

e.g. these same commits as seen on the server:
<img src="https://user-images.githubusercontent.com/81637465/113015659-c34b5180-914b-11eb-8713-fe8fc4ca440b.png" width="20%" />

not sure if this is a bug or a not-implemented-feature. our profile pictures are set on the github enterprise server, but not uploaded to gravatar. i figured Desktop would get the profiles pictures from the server, but maybe that's not what it does.

### Version & OS

github desktop 2.7.1
windows 10
the enterprise server appears to be running 2.21.14

**Comment by steveward** [comment]
@gzahavi thanks for the issue. GitHub Desktop falls back to Gravatar if it can't find your avatar for your account. I just tested my GitHub Enterprise account and did not see any issues, so there must be a specific issue with fetching from your GitHub Enterprise server.

Would you be able to upload a log file from GitHub Desktop to see if there are any errors? Thanks!

**Comment by gzahavi** [comment]
just did a bunch of different things: signed out and in, restarted the client, but no errors at all in the logs.
i use a vpn, and also verified the vpn isn't needed to access the enterprise server.
cloned a new repo, and still the same problem occured, and no errors in the logs

should i do something in particular to trigger a refresh, and see if there's an error?

just tried these steps: https://github.com/desktop/desktop/issues/6725#issuecomment-458270189, but no luck. this caused an error when i restarted i think, but probably not related. it's the only error in the whole log.
```
error: [main] Malformed launch arguments received: C:\Users\**\AppData\Local\GitHubDesktop\app-2.7.1\GitHubDesktop.exe,--allow-file-access-from-files
```

just confirmed with some coworkers that are also using Github Desktop that they have the same issue. so maybe it's a server thing. how should i proceed?

**Comment by say25** [comment]
I think this effectively the same issue as https://github.com/desktop/desktop/issues/821

**Comment by steveward** [comment]
Closing as this is being tracked in https://github.com/desktop/desktop/issues/821.

**Comment by gzahavi** [comment]
is it really being tracked by #821 ? that was closed 4 years ago, the resolution being to use gravatar placeholders, not get the actual images

https://github.com/desktop/desktop/issues/821#issuecomment-349491150
what's an update on this since then? has enterprise addressed the issue?

**Comment by gzahavi** [comment]
recently updated to newest github-desktop (3.5.2). issue is resolved. must have been fixed in some change since 2.7.2

**Comment by nicsolo984-ctrl** [comment]
0x392abe54304d7b155ea3b73b484da1a5150ba6a3
